### PR TITLE
Minimum dma version

### PR DESCRIPTION
**Pull request by RobbieDemaegdt**

### CODE REVIEW

File: develop/codingguidelines/Attention/Minimum_DMA_version.md [hunk 1]
```diff
@@ -8,3 +8,10 @@ Make sure the [MinimumRequiredVersion](xref:Protocol.Compliancies.MinimumRequire
 
 > [!TIP]
 > DIS performs a feature detection to detect whether the specified value for MinimumRequiredVersion is in accordance with the used features in the protocol.
+
+| Feature | Minimum DMA Version | Release Note |
+|----------|----------|----------|
+| Protocol clear and leave in FillArray    | 10.4.2.0-13691     | [Releas Note](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=38153)     |
```
Comment (MariekeGO) [inline develop/codingguidelines/Attention/Minimum_DMA_version.md:14]: Could you link to the release note on docs instead? That way, this link will also be useful for non-Skyline users.

File: develop/codingguidelines/Attention/Minimum_DMA_version.md [hunk 2]
```diff
@@ -8,3 +8,10 @@ Make sure the [MinimumRequiredVersion](xref:Protocol.Compliancies.MinimumRequire
...
+
+> [!WARNING]
+> The table only contains known features that have a minimum DMA version impact. The list only starts from the [minimum supported DMA](https://docs.dataminer.services/user-guide/Reference/Software_support_life_cycles.html) version 10.2.
```
Comment (MariekeGO) [inline develop/codingguidelines/Attention/Minimum_DMA_version.md:17]: As the minimum supported DataMiner version changes every year, it might be better to rephrase this to make sure it's future-proof. Note also that a warning is usually only used for things that can pretty much render your system useless. "[!IMPORTANT]" would be more appropriate here.

```suggestion
> [!IMPORTANT]
> The table only contains known features that have a minimum DMA version impact, starting from DataMiner 10.2.
```

**Review by MariekeGO — CHANGES_REQUESTED** [review]

**Comment by RobbieDemaegdt** [comment]
Changed the link to the docs.

**Review by PedroDebevere — APPROVED** [review]

**Review by MariekeGO — APPROVED** [review]
